packages/mui-material/src/Popover/Popover.js

```diff
@@ -99,7 +99,7 @@ const Popover = React.forwardRef(function Popover(inProps, ref) {
     action,
     anchorEl,
     anchorOrigin = {
-      vertical: 'top',
+      vertical: 'bottom',
```

Comment:
This is a breaking change, we cannot change it, also it doesn't look like it fixes the issue linked. Let's discuss some potential fixes on the issue first, before opening a PR :)